### PR TITLE
Improve event administrate table

### PR DESCRIPTION
**Pull request by juniwbjerde**
Fixes https://github.com/webkom/lego/issues/1993

Added: 
- column with a number for every attendee (the number is the index of the attendee in the array of all registered attendees, ie. the numbers will be arranged so that number 1 is the first who signed up for the event, number 2 is the second and so on)
- column with the pool of the attendee
- ability to choose which column to show given a set of possibilities (for example i have used this for the columns grade and pool, because they're normally the same (so you don't want to show them both), but in the few cases when they aren't it's nice to be able to choose which one you want to see)
- ability to search in the name column (functionality was already implemented, just added the ability for this specific column)
- ability to sort the table based on a chosen column (first click sorts ascending, second sorts descending) 

(don't know if i can add example-photos to github, as an example from staging would contain personal info about users... so i hope this illustrates the concepts well enough)

Table header with all the functions (search, sort, choose column, and new coulmns)
image

Choose between pool and grade:
image

Nr. column and example of sorting:
image
image

### CODE REVIEW

File: app/components/Form/RadioButton.js [hunk 1]
```diff
@@ -30,7 +30,6 @@ function RadioButton({
         checked={inputValue === value}
         type="radio"
         id={id}
-        value={inputValue}
```
[the remark this answers is not in the document]
Comment (juniwbjerde) [reply app/components/Form/RadioButton.js:33]: Hmm.. I can't remember why i did this now. Doesn't seem like there's a good reason to remove it to me, so i'll change it back.

File: app/components/Table/index.js [hunk 2]
```diff
@@ -109,6 +137,25 @@ export default class Table extends Component<Props, State> {
...
+      { showColumn: { [dataIndex]: columnIndex } }, 
+      () => this.onChange()
+    );
+  };
+
+  onSortInput = (dataIndex: any, sorter: any) => {
+    let direction = 'asc';
+    if (this.state.sort.dataIndex == dataIndex){
+      if (this.state.sort.direction == 'asc') direction = 'desc';
+    }
```
[the remark this answers is not in the document]
Comment (juniwbjerde) [reply app/components/Table/index.js:151]: No i agree, it looks way better! I'll fix

**Comment by LudvigHz** [comment]
@juniwbjerde Could you fix the lint/flow issues so we can get this merged? :smile:

**Comment by LudvigHz** [comment]
@juniwbjerde We can merge this right? This has been sitting here for a while now I see :sweat_smile:.

**Comment by juniwbjerde** [comment]
@LudvigHz yeah, go ahead!

**Pull request merged**